client/src/docs/command-docs.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2019. Beautiful Code BV, Rotterdam, Netherlands
+ * Licensed under GNU GENERAL PUBLIC LICENSE Version 3.
+ */
+
+import * as React from "react"
+import { CardText } from "reactstrap"
+
+import { Command } from "../state/app-state"
+
+export interface ICommandDoc {
+    title: Command,
+    body: JSX.Element
+}
+/*
+    Start = "Start",
+    Stop = "Stop",
+    Terraform = "Terraform",
+
+ */
+export const COMMAND_DOCS: ICommandDoc[] = [
+    {
+        title: Command.AbandonFix,
+        body:
+            <CardText>
```

Comment:
Also you should just return text, or a generic `span`/`div` if you need to. Don't wanna lock yourself into a `CardText`

Reply:
Sure but that's one find-replace.